### PR TITLE
[IT-1552]  Apply CostCenter tags to resources

### PR DESCRIPTION
**Pull request by zaro0508**
Add a [Sage CostCenter tag](https://github.com/Sage-Bionetworks/aws-infra/tree/master/templates/tags) to resources deployed by the sandbox
provisioner using [sceptre stack_tags](https://docs.sceptre-project.org/dev/docs/stack_config.html#stack-tags)

Also change sceptre configs to use sceptre [template handlers](https://docs.sceptre-project.org/dev/docs/template_handlers.html) instead of hooks

### CODE REVIEW

**Review by brucehoff — APPROVED** [review]

File: config/prod/bgrande-nextflow-service-account.yaml [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
-template_path: "remote/service-account.yaml"
+template:
+  type: "http"
+  url: "https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/aws-infra/v0.2.7/IAM/service-account.yaml"
 stack_name: "bgrande-nextflow-service-account"
 stack_tags:
   Department: "CompOnc"
   Project: "imCORE"
   OwnerEmail: "bruno.grande@sagebase.org"
+  CostCenter: "Mobile Toolbox Project Core / 112501"
```
Comment (brucehoff) [inline config/prod/bgrande-nextflow-service-account.yaml:10]: How did you determine this cost center?  I guessed a different one for this project, as shown in this mapping: https://github.com/Sage-Bionetworks/aws-cost-accounting/blob/main/project_to_cost_center.tsv

File: config/prod/tom-ncats.yaml [hunk 2]
```diff
@@ -1,6 +1,13 @@
...
-template_path: "remote/managed-ec2-linux-v1.j2"
+template:
+  type: "http"
+  url: "https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/aws-infra/v0.2.7/managed-ec2-linux-v1.j2"
 stack_name: "tom-ncats"
+stack_tags:
+  Department: "CompOnc"
+  Project: "dream"
+  OwnerEmail: "thomas.yu@sagebase.org"
+  CostCenter: "Genie-AACR / 312000"
```
Comment (brucehoff) [inline config/prod/tom-ncats.yaml:10]: How did you determine this CostCenter?  I can't infer any particular CostCenter from "dream".

File: config/prod/bgrande-nextflow-service-account.yaml (hunk 1 above)
Comment (zaro0508) [reply config/prod/bgrande-nextflow-service-account.yaml:10]: just a guess.  @BrunoGrandePhD can you please verify the most appropriate value for this?

File: config/prod/tom-ncats.yaml (hunk 2 above)
Comment (zaro0508) [reply config/prod/tom-ncats.yaml:10]: just a guess.  @thomasyu888  can you please verify the most appropriate value for this?

Comment (zaro0508) [reply config/prod/tom-ncats.yaml:10]: ping @thomasyu888

Comment (brucehoff) [reply config/prod/tom-ncats.yaml:10]: @zaro0508 I think this is fine.

**Review by brucehoff — APPROVED** [review]

File: config/prod/bgrande-nextflow-service-account.yaml (hunk 1 above)
Comment (zaro0508) [reply config/prod/bgrande-nextflow-service-account.yaml:10]: will remove with a  follow on PR.

Comment (zaro0508) [reply config/prod/bgrande-nextflow-service-account.yaml:10]: removed in PR https://github.com/Sage-Bionetworks/sandbox-provisioner/pull/402

**Pull request merged**